packages/components/src/Divider/Divider.css

```diff
@@ -0,0 +1,10 @@
+.divider {
+  height: var(--space-minuscule);
+  margin-bottom: var(--space-large);
```

Comment:
I wonder if divider should define any margin or just be tiny and expected to be within a `<Content>` or something… 🤔